misc/python/materialize/mzbuild.py

```diff
@@ -208,21 +212,26 @@ def __init__(self, rd: RepositoryDetails, path: Path, config: Dict[str, Any]):
         self.strip = config.pop("strip", True)
         self.extract = config.pop("extract", {})
         self.rustflags = config.pop("rustflags", [])
+        self.channel = config.pop("channel", None)
         if rd.coverage:
             self.rustflags += [
                 "-Zinstrument-coverage",
-                "-Clink-dead-code",
                 # Nix generates some unresolved symbols that -Zinstrument-coverage
```

Comment:
This was previously required to be present to get coverage reports, now it is required to not be present ...